src/emcpy/plots/create_plots.py

```diff
@@ -315,67 +391,198 @@ def set_yticklabels(self, labels=list(),
                              'len of yticks. Set yticks appropriately ' +
                              'or change labels to be len of yticks.')
 
-    def add_colorbar(self, label=None, orientation='horizontal',
-                     label_fontsize=12, extend='neither'):
+
+class CreateMap(EMCPyPlots):
+    """
+    Creates a map axes on a figure to plot data.
+    """
+
+    def __init__(self, fig=plt.figure(figsize=(12, 8)),
```

Comment:
Is there a reason, `fig=plt.figure(figsize=(12,8))` not be in the `EMCpyPlots` constructor?
Same for `CreatePlots`.
It seems an appropriate place for initializing a figure.